src/Kanbas/Courses/Quizzes/Preview/Question/index.tsx

```diff
@@ -1,7 +1,63 @@
-function QuizQuestionPreview() {
+import { useEffect, useRef } from "react";
+import { IKanbasQuizQuestion } from "../../../../store/interfaces/quizzes";
+import { FaFlag } from "react-icons/fa";
+import QuizQuestionHeader from "./Common/Header";
+import MultipleChoice from "./Common/MultipleChoice";
+import { eQuizQuestionType } from "../../../../store/enums/eQuizQuestionType";
+import ShortAnswers from "./Common/ShortAnswers";
+import "./index.css";
+
+function QuizQuestion({
+    question,
+    index,
+    flaggedQuestions,
+    handleFlagClick,
+    setRef }:
+    {
+        question: IKanbasQuizQuestion,
+        index: number,
+        flaggedQuestions: boolean[],
```

Comment:
Why does a list of flagged questions need to be passed here? Instead, pass a method `isQuestionFlagged` from parent, and use that to get this info using parent's state.